Configs/en-de_captions.yml

```diff
@@ -48,12 +48,12 @@ num_distractors_train: 15
 num_distractors_valid: 15
 
 # Loading and saving data:
-train_captions: ./DataLink/captioning_new/en_captions_train.jsonl
-valid_captions: ./DataLink/captioning_new/en_captions_val.jsonl
-train_images: ./DataLink/captioning_new/images_train.pt
-valid_images: ./DataLink/captioning_new/images_val.pt
+train_captions: ./Data/captioning_new_new/en_captions_train.jsonl
```

Comment:
Similar comment as above: Should these remain `DataLink` since `Data` may be empty/are not uploaded to GitHub?